daemon/src/wire.rs

```diff
@@ -159,6 +159,20 @@ impl TakerToMaker {
             TakerToMaker::Unknown => "TakerToMaker::Unknown",
         }
     }
+
+    pub fn order_id(&self) -> Option<OrderId> {
+        use TakerToMaker::*;
+        match self {
+            DeprecatedTakeOrder { order_id, .. }
+            | TakeOrder { order_id, .. }
+            | ProposeRollover { order_id, .. }
+            | ProposeRolloverV2 { order_id, .. }
+            | Protocol { order_id, .. }
+            | RolloverProtocol { order_id, .. }
+            | Settlement { order_id, .. } => Some(*order_id),
+            _ => None,
```

Comment:
🛠 Please don't use `_` here - if we add a new message with `order_id` we would ignore the `order_id`, if you mention all choices we will notice when adding the new message to the match.